src/System/Led.cpp

```diff
@@ -5,57 +5,57 @@ Led::Led(uint16_t pin, uint8_t brightness){
     _brightness = brightness;
     // Enable output pin
     pinMode(pin, OUTPUT);
-    _state = OFF;
```

Comment:
Changed this enum as it was causing conflicts with constants that have since been removed. Better to have a more descriptive enum anyway.